nixos/modules/services/web-apps/comfyui.nix

```diff
@@ -0,0 +1,139 @@
+{ config, lib, options, pkgs, ...}:
+
+with lib;
```

Comment:
to address #208242 for this module, uses of with such as this should be avoided.
consider `inherits` where you find yourself using `lib.foo` excessively.

Reply:
Indeed, `with lib;` needs to be removed.